Mathlib/Data/Nat/Fib/Zeckendorf.lean

```diff
@@ -119,9 +119,9 @@ def zeckendorf : ℕ → List ℕ
   have := zeckendorf_aux (n.succ_pos.trans_eq $ Eq.symm ‹_›)
   a :: zeckendorf (m - fib a)
 
-@[simp] lemma zeckendorf_zero : zeckendorf 0 = [] := rfl
+@[simp, eqns] lemma zeckendorf_zero : zeckendorf 0 = [] := rfl
```

Comment:
I don't think this is how this attribute works: aren't you supposed to give it arguments, and attach it to the def?

Reply:
Tried it but it still seem to be a no-op.